NEWS.md

```diff
@@ -1,63 +1,57 @@
-3.14.1
+3.15.1
 
-<!-- Fri, 23 Feb 2018 09:08:05 +0100 -->
+<!-- Thu, 19 Apr 2018 10:55:16 +0200 -->
 
 ## Features
 
- * Password protected ssl keys (#1888)
- * Add OpenBSD module to system() source (#1875)
- * Add Ubuntu Trusty support to Docker build (#1849)
- * Add filters as selectors in contextual data (#1838)
+ * Support added for `if`/`elif`/`else` blocks to the configuration file syntax.
+   (#1856)
+ * Dramatically improved debug messages during filter/parser evaluation. (#1898)
+ * The `hook-commands` module was added, allowing one to run custom commands on
+   source or destination setup and teardown. (#1951)
+ * Implemented a way to skip processing included config file snippets in case a
+   dependency is missing: The `@requires json` pragma. (#827, #1956)
+ * Basic client-side failover support was implemented. (#1905)
+ * Errors from python destinations are now reported together with any exception
+   text (if any). (#1931)
```

Comment:
Do we want to include #1911?

Reply:
@MrAnno: yes, we want

Reply:
ack, added.

Will force-push the branch once I'm done with the rest too.

Reply:
Thanks :+1: